Sources_v3/StreamChatUI/Utils/Animation.swift

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+func WithAnimation(_ actions: @escaping () -> Void, completion: ((Bool) -> Void)? = nil) {
```

Comment:
nit: Are you happy with the name? It seems I don't actually like it 🤦‍♂️ Maybe `Animate { }` would read better on the caller side?